### PR TITLE
Add comment field

### LINKED ISSUE #68: Add comments element
This element should be able to be configured at least with following parameters:

1. Font height
2. Font weight
3. Font color

### PR DESCRIPTION
**Pull request by IgorZiegler**
Config example:
```
"comment": {
    "block": "comment",
    "text": "Если поля вашей квитанции отличаются от представленных на экране, добавьте услугу, предварительно удалив ненужную. Максимальное количество полей - 6.",
    "fontColor": "#252525",
    "fontSize": "86%"
}
```

Add example to modal and array json files

fixes #68

### CODE REVIEW

File: frontend/blocks/comment/comment.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import template from './comment.jade';
+import './comment.scss';
+
+import Base from '../base/base';
+import Factory from '../factory';
+
+export default class Comment extends Base {
+
```
Comment (102) [inline frontend/blocks/comment/comment.js]: Why do you add this empty line?

File: frontend/blocks/comment/comment.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  get templateFn() {
+    return template;
+  }
+
```
Comment (102) [inline frontend/blocks/comment/comment.js]: Why do you add this empty line?

File: frontend/blocks/array/array_field.js [hunk 3]
```diff
@@ -37,7 +38,7 @@ export default class ArrayField extends Container {
         labelWidth: config.itemLabel ? 3 : 0,
         layout: items.map(() => ({ count: 3, width: [1, 8, 3] })),
         items: fieldsetItems
-      }, {
+      }, config.comment, {
```
Comment (102) [inline frontend/blocks/array/array_field.js]: place `config.comment` on a separate line

File: public/examples/forms/array.json [hunk 4]
```diff
@@ -16,6 +16,12 @@
         "glyphicon": "trash",
         "cls": "btn-link"
       },
+      "comment": {
+        "block": "comment",
+        "text": "Если поля вашей квитанции отличаются от представленных на экране, добавьте услугу, предварительно удалив ненужную. Максимальное количество полей - 6.",
+        "fontColor": "#252525",
+        "fontSize": "86%"
```
Comment (102) [inline public/examples/forms/array.json]: Why do we want to have both `.small` with `font-size: 86%` and `fontSize` inline style with `86%` in our example?
Also, may be we want to have some default value for this properties?

**Comment by 102** [comment]
This block doesn't looks well when placed as comment inside array (something went wrong with left margin)_

File: public/examples/forms/array.json (hunk 4 above)
Comment (IgorZiegler) [inline public/examples/forms/array.json]: I have remove scss with .small style. And now we have only fontSize inline style

**Pull request merged**